vllm/transformers_utils/tokenizer.py

```diff
@@ -88,6 +88,9 @@ def get_tokenizer(
                 "Cannot use the fast tokenizer in slow tokenizer mode.")
         kwargs["use_fast"] = False
 
+    if "truncation_side" not in kwargs:
+        kwargs["truncation_side"] = "left"
```

Comment:
This will also affect the tokenizers in the `LLMEngine`. Would that break anything?

Reply:
No, it only has an effect if `truncate=True` is also passed when tokenizing. And for causal LMs I don't think it would ever make sense to right-truncate input anyhow.